client/my-sites/stats/site.jsx

```diff
@@ -109,7 +110,10 @@ class StatsSite extends Component {
 			<Main wideLayout={ true }>
 				<StatsFirstView />
 				<SidebarNavigation />
-				<StatsNavigation section={ period } />
+				<StatsNavigation
+					section={ period }
```

Comment:
weird little semantic wrinkle here: if `this.props.section` exists, it would overwrite the `section` you're passing explicitly. If you want to guarantee that your `section` will win, you want to write this `{ ...this.props } section={ period}`.